packages/mux-player/src/template.ts

```diff
@@ -152,6 +164,24 @@ const MediaFullscreenButton = () => html`
   ${icons.FullscreenExit()}
 </media-fullscreen-button>`;
 
+// prettier-ignore
+export const VodChromeExtraSmall = (props: MuxTemplateProps) => html`
```

Comment:
non-blocking: PIP/Fullscreen buttons weren't discussed, but I'm cool with adding them and iterating as this hasn't gone through design and is easy enough to update.

Reply:
yes seemed useful if people want to see the tiny video in a larger view